demo/Dockerfile.jupyter

```
@@ -0,0 +1,32 @@
+FROM jupyter/scipy-notebook:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)